### ISSUE TITLE
Including blank page break markers

### ISSUE THREAD

**Issue by mattgarrish** [w3c/epub-specs#1501]
Is it required to include page break markers for blank pages in the print to meet the accessibility requirements?

If so, we should probably include a technique for how to include and label these pages.

**Comment by clapierre** [comment]
If a publication has pages and in the Metadata it says that the source of the pages is from a physical book which actually has blank pages inside it for layout reasons presumably wouldn't we also require the digital book to match the print book?  Does just omitting the page # perform the same function in digital as a physical blank page does in print?

**Comment by mattgarrish** [comment]
Blank pages are just a quirk of print so that new chapters/sections begin on a left/right hand page. They have no real content value in a digital work beyond a measure of completeness that every page has been faithfully reproduced no matter if it contains nothing.

Given that actual missing pages of content would be a serious problem for every reader, are they more bother than they're worth?

As I mentioned in #1471, you could always mention in the summary that blank pages have been omitted if there's concern about possible confusion.

The one thing they do is ensure that if a user plugs in a random number in a "go to" function they'll end up at a page.

**Comment by GeorgeKerscher** [comment]
A student would never be told to turn to a blank page. However, in the sequencing in print books, blank pages have a number that is not printed, e.g. page 69 has text, page 70 is blank, and page 71 has text, the page is counted in total pages.

**Comment by mattgarrish** [comment]
> the page is counted in total pages

Sure, but even if we require blank pages we still have the problem that the digital edition may not include all the same content as the print.

Do publishers have to include page markers for pages that haven't been reproduced that say the page has been omitted from the digital version? That's a related question to this one. If not, should we be requiring blank pages be present?

I'm not saying we should mandate the markers be omitted, only questioning here whether not including them for blank pages or omitted content constitutes a failure.

**Comment by mattgarrish** [comment]
Closing this issue as the discussion in #1502 has already moved into this topic. I'll rename that issue to cover both blank and omitted pages.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2021-03-11-epub-a11y#section4) on 2021-03-11

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

### 4. pages related issues 

_See github issue [#1503](https://github.com/w3c/epub-specs/issues/1503), [#1502](https://github.com/w3c/epub-specs/issues/1502), [#1501](https://github.com/w3c/epub-specs/issues/1501), [#1500](https://github.com/w3c/epub-specs/issues/1500), [#1542](https://github.com/w3c/epub-specs/issues/1542)._



**Matt Garrish:**: Updated EPUB accessibility for addressing a part of the following static pages issues. Some requirements can be addressed by EPUB Accessibility while others should go to best practices.

> *Avneesh Singh:* [https://w3c.github.io/epub-specs/epub33/a11y/index.html#sec-page-nav](https://w3c.github.io/epub-specs/epub33/a11y/index.html#sec-page-nav)

**Matt Garrish:** we had different questions about pagelists  
… one propose is to structure the requirements for pagelists  
… e.g. level A: some pages in the pagelist  
… level AA: all content pages linked in pagelist  
… level AAA: all pages (no exclusion, even blank pages)  
… I didn't move over with my requirements  

**Avneesh Singh:** I don't think these issues can be added in the guidelines, maybe the best place is best practices  

**Tzviya Siegman:** we had similar in PDFs with "this page intentionally left blank", where we left blank pages for starting the chapters on page right  
… I think that marking blank pages can be really confusing  
… maybe we can use ARIA  

**Avneesh Singh:** yes, but then someone will say that ARIA is not only for AT  

**Charles LaPierre:** I'm not sure if Read Aloud by the Reading system can get that information  

**Matt Garrish:** for sure we can investigate the use of ARIA label or something similar  

**Avneesh Singh:** I think this task force should focus on the requirements Matt put in the issues  
… we can go through the issue tracker  
… and discuss there  

**Matt Garrish:** we have a issue about the ordering of the pagelist  
… if I have the pages moved in the digital version, what happen to the pagelist?  

**Tzviya Siegman:** I have several examples of this  

**Avneesh Singh:** I think for AT users it would be useful to have an alert  

**Charles LaPierre:** maybe we can put in in the accessibility summary (metadata), but do we need to have some requirements for reading systems?  

**Gregorio Pellegrino:** maybe something like play order attribute in NCX tocs  

---


</details>
